Makefile

```diff
@@ -23,6 +31,12 @@ types_easyjson.go: types.go
 test: types_easyjson.go
 	go test -v
 
+.PHONY: lint
```

Comment:
also nice to backport to go template.

Reply:
I'll do that

Reply:
Done via https://github.com/kubewarden/github-actions/pull/64 and https://github.com/kubewarden/go-policy-template/pull/40